public/app/features/transformers/fieldToConfigMapping/FieldToConfigMappingEditor.tsx

```diff
@@ -60,13 +70,26 @@ export function FieldToConfigMappingEditor({ frame, mappings, onChange, withRedu
     }
   };
 
+  const onChangeHandlerArguments = (row: FieldToConfigRowViewModel, handlerArguments: HandlerArguments) => {
+    const existingIdx = mappings.findIndex((x) => x.fieldName === row.fieldName);
```

Comment:
Since it's not reused this could can be placed directly in the _if_ statement below.